WcaOnRails/app/views/delegate_reports/edit.html.erb

```diff
@@ -25,8 +25,8 @@
     <%= f.input :remarks, input_html: { class: "markdown-editor markdown-editor-image-upload" } %>
 
     <%= f.button :submit, class: "btn-primary" %>
-    <% is_actually_posted = DelegateReport.find(@delegate_report.id).posted? %>
-    <% if !is_actually_posted %>
+    <% can_submit_delegate_report = DelegateReport.find(@delegate_report.id).can_submit_delegate_report? %>
+    <% if can_submit_delegate_report %>
```

Comment:
This is where we can keep the check to display the "submit" button only to non-trainee delegates :)